### ISSUE TITLE
Error from GET-COMMAND -PARAMETERNAME THROW 

### ISSUE THREAD

**Issue by yecril71pl** [pester/Pester#1606]
## 1. General summary of the issue

The command ``{ GET-COMMAND -PARAMETERNAME THROW }`` generates a run-time error.
So does the pipeline ``{ GET-COMMAND SHOULD | FOREACH-OBJECT PARAMETERS | FOREACH-OBJECT THROW }``.

## 2. Describe Your Environment

<DL ><DT >Pester version     : <DD >5.0.2 C:\Users\User\Documents\PowerShell\Modules\Pester\5.0.2\Pester.psd1
<DT >PowerShell version : <DD >7.0.2
<DT >OS version         : <DD >Microsoft Windows NT 10.0.18363.0</DL >

## 3. Expected Behavior 

<TABLE ><TR ><TH >CommandType<TH>     Name<TH>                                               Version  <TH >  Source<TR ><TD >Function        <TD >Should                                             <TD >5.0.2      <TD >Pester</TABLE >

## 4.Current Behavior

<DL ><DT >MethodInvocationException: <DD >C:\Users\User\Documents\PowerShell\Modules\Pester\5.0.2\Pester.psm1:6403</DD >
<TABLE ><TR ><TD >
Line 
6403 <TD >          <CODE >$myLine = # we can use $myInvocation.Line to get the surround</CODE > …
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      Exception calling "<CODE >Substring</CODE >" with "1" argument(s): "<CODE >StartIndex</CODE > cannot be less than zero. (Parameter
      '<CODE >startIndex</CODE >')"</TABLE >


<TABLE ><TR ><TH >CommandType<TH>     Name<TH>                                               Version  <TH >  Source<TR ><TD >Function        <TD >Should                                             <TD >5.0.2      <TD >Pester</TABLE >

## 5. Possible Solution

<!--

Have a solution in mind? Bug fix pull requests are always welcome.

https://github.com/pester/Pester/wiki/Contributing-to-Pester has detailed instructions on how to contribute.

If you are proposing a feature, let's discuss it here first.

-->
We cannot assume that we can use ``$myInvocation.Line`` to get the surrounding context.
  We should handle the case when we cannot.
## 6. Context

<!--

How has this issue affected you? What are you trying to accomplish?

-->

I was trying to get the description for the THROW parameter.

**Comment by nohwnd** [comment]
@StartAutomating would you have a look please?

**Comment by fflaten** [comment]
Error fixed in #1804 (available in 5.1.1)
